usautobuild/actions/api_caller.py

```diff
@@ -39,4 +36,4 @@ def version_to_date(version: str) -> str:
         month = version[2:4]
         day = version[4:6]
 
-        return f"20{year}-{month}-{day}"
+        return f"20{year:0>2}-{month:0>2}-{day:0>2}"
```

Comment:
What's this

Reply:
1 will become 01

Reply:
It wasn't doing that already?

Reply:
right, inputs are strings already, not numbers. i will undo this